src/encoded/static/components/regulome_search.js

```diff
@@ -593,26 +593,34 @@ class RegulomeSearch extends React.Component {
     }
 
     updateDimensions() {
-        const screenWidth = this.applicationRef.offsetWidth;
-        let thumbnailWidth = 0;
-        if (screenWidth > 865) {
-            thumbnailWidth = (this.applicationRef.offsetWidth / 3) - 40;
-        } else {
-            thumbnailWidth = this.applicationRef.offsetWidth - 40;
+        // check for applicationRef because otherwise there are errors during resizing process
+        if (this.applicationRef) {
+            const screenWidth = this.applicationRef.offsetWidth;
+            let thumbnailWidth = 0;
+            if (screenWidth > 787) {
+                thumbnailWidth = (this.applicationRef.offsetWidth / 3) - 40;
+            } else if (screenWidth > 483) {
+                thumbnailWidth = (this.applicationRef.offsetWidth / 2) - 40;
+            } else {
```

Comment:
By embedding these numbers like "787" and "483" in the code, could this cause a broader problem later if multiple places in the code need the same breakpoints, or if someone else needs to know what these widths are for? Maybe we need something sort of like the $screen-sm-min constants, but in JS.